src/site/tools/observatory/cpu-profile.markdown

```diff
@@ -196,6 +196,12 @@ VM
 None
 : Hides all tags. The functions are displayed from most CPU intensive to least.
 
+## Stubs
+
+### InlineCacheStub / SubtypeTestCacheStub
```

Comment:
Please list each stub that a user could encounter and include a brief description of what it means.